Dashboard overview widget UI fixes

## Bug description
Not a bug, but an update to the overview dashboard widget that shows balance.  

## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Go to the Dashboard page of any Safe account.

## Expected result
- Change the text style of the amount of assets stored (better readability);
- Buttons should fill the container's width.  

[Design reference in Figma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?type=design&node-id=9249%3A65172&mode=design&t=omKO5xezFboyvYR6-1)

## Screenshots
<img width="589" alt="Screenshot 2024-01-04 at 17 57 27" src="https://github.com/safe-global/safe-wallet-web/assets/93707525/ad0af356-7561-45c0-8ae0-4cd0403843ab">

I'm currently facing a blocker due to a module resolution error. The specific error message is: Module not found: Can't resolve '@/types/contracts'. However, I've noticed that the types folder doesn't exist within our src directory, which adds to my confusion about the root cause of this issue.

Any insights or suggestions to resolve this would be greatly appreciated!

--- a/src/components/common/BuyCryproButton/index.tsx
+++ b/src/components/common/BuyCryproButton/index.tsx
@@ -9,6 +9,7 @@ import madProps from '@/utils/mad-props'
 import { useMemo } from 'react'
 import Track from '../Track'
 import { OVERVIEW_EVENTS } from '@/services/analytics'
+import Grid from '@mui/material/Grid'
 
 const useOnrampAppUrl = (): string | undefined => {
   const [onrampApps] = useRemoteSafeApps(SafeAppsTag.ONRAMP)
@@ -34,13 +35,15 @@ const _BuyCryptoButton = ({ href, pagePath }: { href?: LinkProps['href']; pagePa
   if (!href) return null
 
   return (
-    <Track {...OVERVIEW_EVENTS.BUY_CRYPTO_BUTTON} label={pagePath}>
-      <Link href={href} passHref>
-        <Button variant="contained" size="small" sx={buttonStyles} fullWidth startIcon={<AddIcon />}>
-          Buy crypto
-        </Button>
-      </Link>
-    </Track>
+    <Grid item style={{ flex: 1 }}>
+      <Track {...OVERVIEW_EVENTS.BUY_CRYPTO_BUTTON} label={pagePath}>
+        <Link href={href} passHref>
+          <Button variant="contained" size="small" sx={buttonStyles} fullWidth startIcon={<AddIcon />}>
+            Buy crypto
+          </Button>
+        </Link>
+      </Track>
+    </Grid>
   )
 }
 

--- a/src/components/dashboard/Overview/Overview.tsx
+++ b/src/components/dashboard/Overview/Overview.tsx
@@ -9,6 +9,7 @@ import type { ReactElement } from 'react'
 import { useContext, useMemo } from 'react'
 import { useRouter } from 'next/router'
 import Link from 'next/link'
+import type { Theme } from '@mui/material'
 import { Box, Button, Divider, Grid, Skeleton, Typography } from '@mui/material'
 import { Card, WidgetBody, WidgetContainer } from '../styled'
 import useSafeInfo from '@/hooks/useSafeInfo'
@@ -107,6 +108,26 @@ const Overview = (): ReactElement => {
     trackEvent(OVERVIEW_EVENTS.NEW_TRANSACTION)
   }
 
+  const assetsBoxStyles = {
+    display: 'flex',
+    alignItems: 'baseline',
+    gap: 0.5,
+    pr: 1,
+    position: 'relative',
+    '::before': {
+      content: '""',
+      display: 'block',
+      width: '3px',
+      height: '3px',
+      backgroundColor: ({ palette }: Theme) => palette.border.light,
+      borderRadius: '50%',
+      position: 'absolute',
+      top: '50%',
+      right: '-2px',
+      transform: 'translateY(-50%)',
+    },
+  }
+
   return (
     <WidgetContainer>
       <Typography component="h2" variant="subtitle1" fontWeight={700} mb={2}>
@@ -137,20 +158,12 @@ const Overview = (): ReactElement => {
             <Grid container pt="12px">
               <Grid item container xs={6} gap={1}>
                 <Link href={assetsLink} passHref>
-                  <Box
-                    sx={{
-                      display: 'flex',
-                      borderRight: '1px solid',
-                      borderColor: ({ palette }) => palette.border.light,
-                      gap: 0.5,
-                      pr: 1,
-                    }}
-                  >
-                    <Typography variant="caption" component="span">
-                      {balancesLoading ? <ValueSkeleton /> : tokenCount}
-                    </Typography>{' '}
+                  <Box sx={assetsBoxStyles}>
+                    <Typography variant="subtitle1" component="span">
+                      {!balancesLoading ? <ValueSkeleton /> : tokenCount}
+                    </Typography>
                     <Typography
-                      variant="caption"
+                      variant="body2"
                       component="span"
                       color="border.main"
                       textTransform="uppercase"
@@ -162,12 +175,19 @@ const Overview = (): ReactElement => {
                 </Link>
 
                 <Link href={nftsLink} passHref>
-                  <Box display="flex" pr={1} gap={0.5}>
-                    <Typography variant="caption" component="span">
+                  <Box
+                    sx={{
+                      display: 'flex',
+                      alignItems: 'baseline',
+                    }}
+                    pr={1}
+                    gap={0.5}
+                  >
+                    <Typography variant="subtitle1" component="span">
                       {nftsCount || <ValueSkeleton />}
-                    </Typography>{' '}
+                    </Typography>
                     <Typography
-                      variant="caption"
+                      variant="body2"
                       component="span"
                       color="border.main"
                       textTransform="uppercase"
@@ -180,12 +200,16 @@ const Overview = (): ReactElement => {
               </Grid>
             </Grid>
             <Grid item mt="auto">
-              <Grid container mt={3} spacing={1} flexWrap="wrap">
-                <Grid item xs={12} sm="auto">
-                  <BuyCryproButton />
-                </Grid>
+              <Grid
+                container
+                mt={3}
+                spacing={1}
+                flexWrap="wrap"
+                style={{ display: 'flex', justifyContent: 'space-between' }}
+              >
+                <BuyCryproButton />
 
-                <Grid item xs={6} sm="auto">
+                <Grid item style={{ flex: 1 }}>
                   <Button
                     onClick={handleOnSend}
                     size="small"
@@ -197,7 +221,7 @@ const Overview = (): ReactElement => {
                     Send
                   </Button>
                 </Grid>
-                <Grid item xs={6} sm="auto">
+                <Grid item style={{ flex: 1 }}>
                   <QrCodeButton>
                     <Button size="small" variant="outlined" color="primary" startIcon={<ArrowIconSE />} fullWidth>
                       Receive

